subseas_flex_fcst/layout.py

```diff
@@ -28,7 +28,7 @@ def app_layout():
 
     # Initialization
     fcst_mu = predictions.sel_cpt_file(DATA_PATH,CONFIG["forecast_mu_file_pattern"],list(CONFIG["leads"])[0],startDates[-1])
-    center_of_the_map = [((fcst_mu.Y[0]+fcst_mu.Y[-1])/2).values, ((fcst_mu.X[0]+fcst_mu.X[-1])/2).values]
+    center_of_the_map = [((fcst_mu.Y[int(fcst_mu.Y.size/2)].values)), ((fcst_mu.X[int(fcst_mu.X.size/2)].values))]
```

Comment:
I've been trying to build the habit of using the notation `da['X']` rather than `data_array.X`. They're usually equivalent, but if the variable name happens also to be the name of a `DataArray` method, `da['sum']` gets to the variable whereas `da.sum` gets you the method.